README.md

```diff
@@ -8,7 +8,7 @@ This project provides a home to Akka Streams connectors to various technologies,
 Documentation
 -------------
 
-Latest [documentation](http://developer.lightbend.com/docs/alpakka/latest/).
+See [Releases](https://github.com/akka/alpakka/releases) page for the reference and API documentation.
```

Comment:
Hm, I'm not sure. A direct link to latest released documentation is much better. I didn't know that `latest` was master. If there is no support for such `current` symlink in techhub we should open an issue, and until solved we need to update this link manually.

That said, we can add more links here, one to the the Releases and another to the API docs.